### DISCUSSION TITLE
Markdown and basic images

### DISCUSSION THREAD

**Discussion by ThibauldMichel** [community/community#22905] [New to GitHub]
<p>Hello everybody! First time in GitHub, and I am very, very lost even after two or three hours on the guide!</p>
<p>I have understood that every text file (image?) in a repository should be written under a markdown format.</p>
<p>It is written <a href="https://help.github.com/articles/about-writing-and-formatting-on-github/" rel="nofollow noopener">here</a>that there is the text formatting toolbar on every text page.</p>
<p>But it seems not true, as I have not found any toolbar on my &lt;Code&gt; tab when editing a document.</p>
<p>Therefore, I am using ghostwritter to write the text in markdown format.</p>
<p>And it is not working. The text in my document on the &lt;Code&gt; tab is still in markdown format.</p>
<p>How can I visualise/consult the final document?</p>
<p>Thanks a lot for your help!</p>

**Comment by mpboom** [comment, marked as answer]
<p>Did you make sure the document has the extension of </p>
<pre><code>.md
</code></pre>
<p>?</p>

**Comment by ThibauldMichel** [comment]
<p>Hello mpboom,</p>
<p>You are right, it was not the right type of document.</p>
<p>I was filling was in the Code tab, and I was searching to fill documents in the Issue tab.</p>
<p>It was that silly!</p>
